src/main/asciidoc/_chapters/developer.adoc

```diff
@@ -2300,19 +2300,37 @@ message is in their patch and you can use that, but be sure the JIRA ID is at
 the front of the commit message, even if the contributor left it out.
 
 [[committer.amending.author]]
-====== Add Amending-Author when a conflict cherrypick backporting
+====== Use GitHub's "Co-authored-by" when there are multiple authors
 
 We've established the practice of committing to master and then cherry picking back to branches whenever possible, unless
 
 * it's breaking compat: In which case, if it can go in minor releases, backport to branch-1 and branch-2.
 * it's a new feature: No for maintenance releases, For minor releases, discuss and arrive at consensus.
 
-When there is a minor conflict we can fix it up and just proceed with the commit.
```

Comment:
I would leave this one as is instead of removing it. Besides GitHub we still use the patch-based approach, where GitHub's features are not available. Probably having it in two sections, instead of one, would be better.

Reply:
Okay, I will leave this section, add a new section below, and add some explanation why there are 2 approaches.

Reply:
Reading the original discussion ( https://lists.apache.org/thread.html/f00b5f9b65570e777dbb31c37d7b0ffc55c5fc567aefdb456608a042@%3Cdev.hbase.apache.org%3E ) it states to discontinue usage of amending author:
" discontinue use of "Amending-Author" and just make use of "Co-authored-by" "
Even if it's a patch file, anyone can add the Co-authors line.

I'm fine either way, but the discussion and the vote was on discontinuing the amending-author.
What do you think @HorizonNet ?

Reply:
Ok, if it was already discussed it is also fine for me. Then let's discontinue the amending-author.